spring-pulsar/src/test/java/org/springframework/pulsar/listener/ConcurrentPulsarMessageListenerContainerTests.java

```diff
@@ -105,6 +96,17 @@ void basicConcurrencyTesting() throws Exception {
 	@Test
 	@SuppressWarnings("unchecked")
 	void exclusiveSubscriptionMustUseSingleThread() throws Exception {
+		MockEnvironment env = setupMockEnvironment(SubscriptionType.Exclusive);
```

Comment:
**[NIT]** I do like the addition of this aggregate mock holder. I would consider renaming it to something other than "Environment" as that has a strong mental tie to the Spring Environment. You know, naming... the hardest thing to do.

Reply:
I was just thinking about another name and was going for MockComponents but I guess that doesn't really work either 😂

Reply:
Maybe PulsarMock? Sounds very generic but then again it's really just for holding the individual objects

Reply:
Hehe. Maybe `PulsarListenerMockComponents`?